### PR TITLE
Dima/ob-183_check_if_palindrome

### PR DESCRIPTION
**Pull request by Horus1223**
add text.txt file
add tests

### CODE REVIEW

File: Dima/text/check_if_palindrome/check_if_palindrome.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+def check_if_palindrome(palindrome):
+    try:
+        palindrome = palindrome.replace(" ", "")
+        if palindrome == palindrome[::-1]:
+            return 'Yes'
```
Comment (glock3) [inline Dima/text/check_if_palindrome/check_if_palindrome.py]: Return boolean value here

File: Dima/text/check_if_palindrome/tests/test_check_if_palindrome.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    def test_when_check_if_palindrome_input_palindrome(self):
+        palindrome = check_if_palindrome('abccba')
+        expected_result = 'Yes'
+        self.assertEqual(palindrome, expected_result)
+
+    def test_when_check_if_palindrome_input_nonpalindrome(self):
+        palindrome = check_if_palindrome('abcsdf cba')
+        expected_result = 'No'
+        self.assertEqual(palindrome, expected_result)
```
Comment (glock3) [inline Dima/text/check_if_palindrome/tests/test_check_if_palindrome.py:20]: Check boolean value here

File: Dima/text/check_if_palindrome/check_if_palindrome.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+def check_if_palindrome(palindrome):
+    try:
+        palindrome = palindrome.replace(" ", "")
```
Comment (glock3) [inline Dima/text/check_if_palindrome/check_if_palindrome.py]: There is python function for this action
https://www.journaldev.com/23625/python-trim-string-rstrip-lstrip-strip
please use it

**Review by glock3 — CHANGES_REQUESTED** [review]

File: Dima/text/check_if_palindrome/check_if_palindrome.py (hunk 3 above)
Comment (Horus1223) [reply Dima/text/check_if_palindrome/check_if_palindrome.py]: Yes, i know, i`ll use it to remove some special symbols, but i used the replace(" ", "") function to add possibility to check palindrome phrases, not only words in string. I added new test for it.

**Review by glock3 — APPROVED** [review]

**Pull request merged**